karma.conf.js

```diff
@@ -0,0 +1,57 @@
+module.exports = function(config) {
+  config.set({
+    browsers: ['PhantomJS'],
+    frameworks: ['jasmine'],
+    files: [
+      'edx_notifications/static/js/HelloWorld.js',
+      'edx_notifications/static/js/specs/hello-world-spec.js'
+    ],
```

Comment:
yes we could(and will) use patterns like

 // list of files / patterns to load in the browser
        files: [
            {pattern: 'analytics_dashboard/static/vendor/**/*.js', included: false},
            {pattern: 'analytics_dashboard/static/bower_components/**/*.js', included: false},
            {pattern: 'analytics_dashboard/static/bower_components/**/*.json', included: false},
            {pattern: 'analytics_dashboard/static/js/models/**/*.js', included: false},
            {pattern: 'analytics_dashboard/static/js/views/**/*.js', included: false},
            {pattern: 'analytics_dashboard/static/js/utils/**/*.js', included: false},
            {pattern: 'analytics_dashboard/static/js/test/specs/*.js', included: false},
        ],